scapy/layers/dot11.py

```diff
@@ -209,6 +209,12 @@ class RadioTap(Packet):
                    ConditionalField(_RadiotapReversePadField(_dbmField("dBm_AntSignal", -256)), lambda pkt: pkt.present and pkt.present.dBm_AntSignal),  # noqa: E501
                    ConditionalField(_RadiotapReversePadField(_dbmField("dBm_AntNoise", -256)), lambda pkt: pkt.present and pkt.present.dBm_AntNoise),  # noqa: E501
                    ConditionalField(_RadiotapReversePadField(ByteField("Antenna", 0)), lambda pkt: pkt.present and pkt.present.Antenna),  # noqa: E501
+                   # RX Flags
+                   ConditionalField(_RadiotapReversePadField(FlagsField("RXFlags", None, -16, ["res1", "BAD_PLCP", "res2"])),   # noqa: E501
+                                    lambda pkt: pkt.present and pkt.present.RXFlags),  # noqa: E501#
+                   # TX Flags
+                   ConditionalField(_RadiotapReversePadField(FlagsField("TXFlags", None, -16, ["TX_FAIL", "CTS", "RTS", "NOACK"])),   # noqa: E501
```

Comment:
It appears to me there is a fifth value :/


mask | meaning
-- | --
0x0001 | Transmission failed due to excessive retries
0x0002 | Transmission used CTS-to-self protection
0x0004 | Transmission used RTS/CTS handshake
0x0008 | Transmission shall not expect an ACK frame and not retry when no ACK is received
0x0010 | Transmission includes a pre-configured sequence number that should not be changed by the driver’s TX handlers

Reply:
> Another RadioTap test was trying out `b16`.
> 
> Now that you have removed this field, you’ll need to delete that line in the test
You're right, I didn't notice it.

> It appears to me there is a fifth value :/ Please correct me if I got I wrong
Right again, will fix it.